File: desktop-app/src/main/java/entities/FightResult.java
package entities;

import java.time.LocalDate;

public class FightResult {
    private int id;
    private int eventId;
    private int matchId;
    private int fighterRedId;
    private int fighterBlueId;
    private int winnerId;
    private String method;
    private int roundEnded;
    private LocalDate fightDate;
    private String notes;

    public FightResult() {
    }

    public FightResult(int eventId, int matchId, int fighterRedId, int fighterBlueId,
            int winnerId, String method, int roundEnded, LocalDate fightDate, String notes) {
        this.eventId = eventId;
        this.matchId = matchId;
        this.fighterRedId = fighterRedId;
        this.fighterBlueId = fighterBlueId;
        this.winnerId = winnerId;
        this.method = method;
        this.roundEnded = roundEnded;
        this.fightDate = fightDate;
        this.notes = notes;
    }

    public FightResult(int id, int eventId, int matchId, int fighterRedId, int fighterBlueId,
            int winnerId, String method, int roundEnded, LocalDate fightDate, String notes) {
        this.id = id;
        this.eventId = eventId;
        this.matchId = matchId;
        this.fighterRedId = fighterRedId;
        this.fighterBlueId = fighterBlueId;
        this.winnerId = winnerId;
        this.method = method;
        this.roundEnded = roundEnded;
        this.fightDate = fightDate;
        this.notes = notes;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getEventId() {
        return eventId;
    }

    public void setEventId(int eventId) {
        this.eventId = eventId;
    }

    public int getMatchId() {
        return matchId;
    }

    public void setMatchId(int matchId) {
        this.matchId = matchId;
    }

    public int getFighterRedId() {
        return fighterRedId;
    }

    public void setFighterRedId(int fighterRedId) {
        this.fighterRedId = fighterRedId;
    }

    public int getFighterBlueId() {
        return fighterBlueId;
    }

    public void setFighterBlueId(int fighterBlueId) {
        this.fighterBlueId = fighterBlueId;
    }

    public int getWinnerId() {
        return winnerId;
    }

    public void setWinnerId(int winnerId) {
        this.winnerId = winnerId;
    }

    public String getMethod() {
        return method;
    }

    public void setMethod(String method) {
        this.method = method;
    }

    public int getRoundEnded() {
        return roundEnded;
    }

    public void setRoundEnded(int roundEnded) {
        this.roundEnded = roundEnded;
    }

    public LocalDate getFightDate() {
        return fightDate;
    }

    public void setFightDate(LocalDate fightDate) {
        this.fightDate = fightDate;
    }

    public String getNotes() {
        return notes;
    }

    public void setNotes(String notes) {
        this.notes = notes;
    }

    @Override
    public String toString() {
        return "FightResult{id=" + id + ", eventId=" + eventId + ", method='" + method + "', fightDate=" + fightDate
                + "}";
    }
}

File: web-app/smartfight-web/src/Entity/FightResult.php
<?php

namespace App\Entity;

use App\Repository\FightResultRepository;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: FightResultRepository::class)]
#[ORM\Table(name: 'fight_result')]
class FightResult
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(name: 'id', type: 'integer')]
    private ?int $id = null;

    #[ORM\Column(name: 'event_id', type: 'integer', nullable: true)]
    private ?int $eventId = null;

    #[ORM\Column(name: 'fighter_red_id', type: 'integer', nullable: true)]
    private ?int $fighterRedId = null;

    #[ORM\Column(name: 'fighter_blue_id', type: 'integer', nullable: true)]
    private ?int $fighterBlueId = null;

    #[ORM\Column(name: 'winner_id', type: 'integer', nullable: true)]
    private ?int $winnerId = null;

    #[ORM\Column(name: 'method', type: 'string', length: 100, nullable: true)]
    private ?string $method = null;

    #[ORM\Column(name: 'fight_date', type: 'date', nullable: true)]
    private ?\DateTimeInterface $fightDate = null;

    public function getId(): ?int { return $this->id; }
    public function getEventId(): ?int { return $this->eventId; }
    public function setEventId(?int $eventId): static { $this->eventId = $eventId; return $this; }
    public function getFighterRedId(): ?int { return $this->fighterRedId; }
    public function setFighterRedId(?int $fighterRedId): static { $this->fighterRedId = $fighterRedId; return $this; }
    public function getFighterBlueId(): ?int { return $this->fighterBlueId; }
    public function setFighterBlueId(?int $fighterBlueId): static { $this->fighterBlueId = $fighterBlueId; return $this; }
    public function getWinnerId(): ?int { return $this->winnerId; }
    public function setWinnerId(?int $winnerId): static { $this->winnerId = $winnerId; return $this; }
    public function getMethod(): ?string { return $this->method; }
    public function setMethod(?string $method): static { $this->method = $method; return $this; }
    public function getFightDate(): ?\DateTimeInterface { return $this->fightDate; }
    public function setFightDate(?\DateTimeInterface $fightDate): static { $this->fightDate = $fightDate; return $this; }

    public function __toString(): string
    {
        return 'Result #' . $this->id;
    }
}
